charts/kargo/templates/users/service-accounts.yaml

```
@@ -50,8 +44,6 @@ metadata:
     {{- include "kargo.labels" . | nindent 4 }}
   {{- if .Values.api.oidc.viewers.claims }}
   annotations:
-    {{- range $claim, $values := .Values.api.oidc.viewers.claims }}
-    rbac.kargo.akuity.io/claim.{{ $claim }}: {{ quote (join "," $values) }}
-    {{- end }}
+    rbac.kargo.akuity.io/claims: {{ .Values.api.oidc.viewers.claims | toJson | quote}}
   {{- end }}
-{{- end }}
+{{- end }}
```

It's standard practice (rooted in POSIX standards) to end files with a line break, so I'd rather keep it.

```suggestion
{{- end }}

```